src/Action/Definitions/GenerateImageEdit.php

```
@@ -106,412 +107,399 @@ public static function config(): array {
-		if ($prompt === '') {
-			return $this->build_direct_error_response(__( 'Error: Prompt is required.', LineConnect::PLUGIN_NAME ));
+	public function edit_image(
+		$prompt,
+		array $images,
+		$mask = null,
+		$size = '1024x1024',
+		$quality = 'auto',
+		$background = 'auto',
+		$output_format = 'png',
+		$output_compression = 75,
+		$input_fidelity = 'high'
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$input_fidelity'. (undefined)

(UnusedFormalParameter)